### DISCUSSION TITLE
Pop-up window.

### DISCUSSION THREAD

**Discussion by winst0niuss** [getgauge/taiko#2602] [General]
How can I automate popup entry?
Can taiko work with this
?

**Comment by winst0niuss** [comment]


**Reply by NivedhaSenthil** [reply]
you can provide basic authentication details into `goto` like this `goto("https://postman-echo.com/basic-auth",{headers:{'Authorization':'Basic cG9zdG1hbjpwYXNzd29yZA=='}})`

**Reply by NivedhaSenthil** [reply]
please provide sample scenario for us to better answer this... it is little unclear what you are trying to automate here
